src/pkg/cli/configResolution.go

```diff
@@ -0,0 +1,104 @@
+package cli
+
+import (
+	"slices"
+	"sort"
+
+	"github.com/DefangLabs/defang/src/pkg/cli/compose"
+	"github.com/DefangLabs/defang/src/pkg/term"
+	"github.com/compose-spec/compose-go/v2/types"
+)
+
+type configOutput struct {
+	Service string `json:"service"`
+	Name    string `json:"name"`
+	Value   string `json:"value,omitempty"`
+	Source  Source `json:"source,omitempty"`
+}
+
+const configMaskedValue = "*****"
+
+type Source int
+
+const (
+	SourceUnknown Source = iota
```

Comment:
I'm missing `env_file` as a source. In fact, you might have more than 1 env file (using `env_file:` in Compose) and more than 1 Compose file, and it'd be useful to show which file it came from (although I'd consider that a stretch goal, not a req for this first version)